documentation/src/main/asciidoc/index.asciidoc

```diff
@@ -2,60 +2,58 @@
 Written and maintained by the {brandname} community
 :icons: font
 
-Get answers and find help in the {brandname} {infinispanversion} doc set.
+++++
+<script async src="https://cse.google.com/cse.js?cx=017043654585294774899:cdrjpvj5awo"></script>
+<div class="gcse-search"></div>
+++++
 
-Embedding {brandname}::
+[discrete]
+== Get Started
 
-* link:titles/embedding/embedding.html[Using {brandname} as an Embedded Library]
+[unstyled]
+* link:titles/getting_started/getting_started.html[Getting Started with {brandname}]
 
-{brandname} Servers::
+[discrete]
+== Administrator
 
+[unstyled]
 * link:titles/server/server.html[{brandname} Server Guide]
 * link:titles/cli/cli.html[Using the Command Line Interface]
-* link:titles/rest/rest.html[{brandname} REST API Guide]
-
-Operator for Kubernetes and OpenShift::
+* link:titles/upgrading/upgrading.html[Upgrading {brandname}]
+* link:titles/security/security.html[Securing {brandname} ]
+* link:titles/tuning/tuning.html[Tuning {brandname} Performance]
 
-* link:https://infinispan.org/infinispan-operator/master/operator.html[{brandname} Operator Guide: Development]
-* link:https://infinispan.org/infinispan-operator/1.1.x/operator.html[{brandname} Operator Guide: Stable]
 
-Hot Rod Clients::
+[discrete]
+== Developer
 
-* link:titles/hotrod_java/hotrod_java.html[Hot Rod Java Client Guide]
+[unstyled]
+* link:titles/embedding/embedding.html[Embedding {brandname}]
+* link:titles/developing/developing.html[Developing for {brandname}]
+* link:titles/configuring/configuring.html[Configuring {brandname}]
+* link:titles/xsite/xsite.html[Using Cross-Site Replication]
+* link:titles/hotrod_java/hotrod_java.html[Hot Rod Java Clients]
```

Comment:
Can you add querying in the front page?

Reply:
@gustavonalle Based on feedback from Tristan, I moved querying from it's own title into the Developer's Guide. We could add a link to the anchor like this:

link:titles/developing/developing.html#indexing_querying[Indexing and Querying]